src/account/mod.rs

```diff
@@ -207,13 +207,11 @@ impl AccountInitialiser {
 
 pub(crate) fn account_id_to_stronghold_record_id(account_id: &AccountIdentifier) -> crate::Result<[u8; 32]> {
     if let AccountIdentifier::Id(id) = account_id {
-        let decoded = hex::decode(id).map_err(|_| anyhow::anyhow!("account id must be a hex string"))?;
-        let id: [u8; 32] = decoded
-            .try_into()
-            .map_err(|_| anyhow::anyhow!("invalid account id length"))?;
+        let decoded = hex::decode(id)?;
+        let id: [u8; 32] = decoded.try_into().unwrap();
```

Comment:
If this panics, won't the error be `id can't be index` instead of `invalid account id length`?

Reply:
yeah you're right but I'll change this with the stronghold refactor so I just left it like this for now